db/migrate/20251022024557_add_trading_enabled_to_classrooms.rb

```diff
@@ -0,0 +1,5 @@
+class AddTradingEnabledToClassrooms < ActiveRecord::Migration[8.0]
+  def change
+    add_column :classrooms, :trading_enabled, :boolean, default: false, null: false
```

Comment:
This will immediately disable trading for all existing classrooms without warning 🤔